### ISSUE TITLE
External devices folders background color changes.

### ISSUE THREAD

**Issue by OJoshuaCG** [Peltoche/lsd#745]
When I use `lsd` command in external devices (usb or another partition) folders background color changes (only folders).

![image](https://user-images.githubusercontent.com/56763404/190497735-8d8a8334-4660-4738-8a81-01035a4a7d50.png)

![image](https://user-images.githubusercontent.com/56763404/190498056-e2694ca3-dbc2-4963-8805-897c9536649f.png)

- OS: Manjaro Linux
- Shell: zsh 5.9
- Terminal: Kitty
- lsd version: 0.23.0

**Comment by meain** [comment]
Could you paste the output of `echo $LS_COLORS`. It is probably coming from those values.

**Comment by OJoshuaCG** [comment]
It shows nothing.

![image](https://user-images.githubusercontent.com/56763404/190636120-8b2267c2-4ba2-48e3-8cc8-481576b65252.png)

**Comment by OJoshuaCG** [comment]
I have solved it thanks to problem #671.

First, copy `dircolors` to a dotfile

```sh
dircolors -p > ~/.dircolors
```


Edit `~/.dircolors` and change `STICKY_OTHER_WRITABLE` and `OTHER_WRITABLE` values to `DIR` value. 

After that, add in `.zshrc` 

```sh
if [[ -f ~/.dircolors ]] ; then
    eval $(dircolors -b ~/.dircolors)     
elif [[ -f /etc/DIR_COLORS ]] ; then
    eval $(dircolors -b /etc/DIR_COLORS)
fi
```


Or, if you are using bash, add in `.bashrc`

```
eval 'dircolors ~/.dircolors' > /dev/null
```
